thirteen/challenges-13.test.js

```diff
@@ -0,0 +1,246 @@
+'use strict';
+/* global describe, expect, test */
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named firstLetters that takes in an array of strings and returns an array
+// containing only the first letter of each string.
+//
+// For example, ['this is great :)', 'wow', 'whyyyyyy :(', ':)))))'] returns ['t', 'w', 'w', ':'].
+// ------------------------------------------------------------------------------------------------
+
+const firstLetters = (strs) => {
+  // Solution code here...
+  return strs.map(el => el.substring(0,1));
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 2
+//
+// Write a function named findHappiness that takes in an array of strings and returns an array
+// containing only the strings from the input array that contain ":)".
+// e.g. ['this is great :)', 'wow', 'whyyyyyy :(', ':)))))'] => ['this is great :)', ':)))))']
+// ------------------------------------------------------------------------------------------------
+
+const findHappiness = (strs) => {
+  // Solution code here...
+  return strs.filter(el => el.includes(')'));

```

Comment:
this should actually be checking for `:)`  so code should be `return strs.filter(str => str.includes(':)'));`